### PR TITLE
Data generation

### PR DESCRIPTION
**Pull request by davidlira1**
Hey Abraham, check out my code and comment please. Thanks!

### CODE REVIEW

File: database/mongoDB/models.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    },
+    infants: {
+        type: Number,
+        required: true
+    }
+});
+
+const Booking = mongoose.model('Booking', bookingSchema);
+
+module.exports = {
```
Comment (abrahamesparza) [inline database/mongoDB/models.js:77]: Great schema structure!

File: database/mySQL/queries.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ const updateBooking = (bookingId, updatedBooking, callback) => {
   keyVals = keyVals.substring(0, keyVals.length - 1);
   
   var queryStr = `
-  UPDATE BOOKINGS 
+  UPDATE bookings 
    SET ${keyVals}
    WHERE id = ${bookingId}
    `;
```
Comment (abrahamesparza) [inline database/mySQL/queries.js:48]: Your file looks clean and understandable. Only suggestion I have here is perhaps use variable names for the intended action. Example:
getListing: selectQry | postBooking: insertQry | updateBooking: updateQry.

File: database/simulated-dataset/generate-data/generate_bookings.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            }
+        };
+
+        
+    }
+    appendFile(db, fileName, bookingsData);
+    console.log(id - 1);
+    return;
+};
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (abrahamesparza) [inline database/simulated-dataset/generate-data/generate_bookings.js:81]: Well written code! I don't have anything to suggest here, I like that there is comments between each function. Makes code much more easier to read and understand.

File: database/simulated-dataset/load-data/load_to_postgreSQL.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+.then(() => {
+    console.log('Populated listings table with data!');
+    return client.query(
+        `COPY bookings FROM '${__dirname}\\..\\data-csv\\bookings1.txt' WITH CSV HEADER`
+    )
+})
+.then(() => {
+    console.log('Populated bookings table with data (1)!');
+    return client.query(
+        `COPY bookings FROM '${__dirname}\\..\\data-csv\\bookings2.txt' WITH CSV HEADER`
```
Comment (abrahamesparza) [inline database/simulated-dataset/load-data/load_to_postgreSQL.js:70]: This is interesting, does this essentially COPY what is in your bookings2.txt file and load everything in that CSV file WITH CSV HEADER into your database?

File: database/simulated-dataset/generate-data/generate_listings.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  }
+
+  appendFile(db, 'listings', listingData);
+
+  return;
+}
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+module.exports = generateListings;
+
+
```
Comment (abrahamesparza) [inline database/simulated-dataset/generate-data/generate_listings.js:52]: Great job writing clear concise and easy to understand code. I did not have many suggestions for improvements as I did not find any areas that I believe needed improvement (aside from the variable names I suggested in the other file).

File: database/simulated-dataset/load-data/load_to_postgreSQL.js (hunk 4 above)
Comment (davidlira1) [reply database/simulated-dataset/load-data/load_to_postgreSQL.js:70]: yea pretty much lol

File: database/simulated-dataset/generate-data/generate_listings.js (hunk 5 above)
Comment (davidlira1) [reply database/simulated-dataset/generate-data/generate_listings.js:52]: Thank Abraham!

**Pull request merged**